### PR TITLE
Document adding a location

### PR DESCRIPTION
**Pull request by timarney**

### CODE REVIEW

File: docs/location-setup.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+https://the-new-location.rescheduler-dev.cds-snc.ca
+```
+
+4. **Whitelist** the domain
+
+The locations `index page` (/src/locations/index.js) contains a whitelist array of locations
+
+Add the new location to the whitelist
+
+```diff
```
Comment (pcraig3) [inline docs/location-setup.md]: wow, this is so cool 😍

File: docs/location-setup.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+### Adding Locations
+
+1. **Create** a new location (JavaScript) file `the-new-location.js`
+
+2. **Add** it to the locations directory `ircc-rescheduler/web/src/locations/the-new-location.js`
+
+3. The location file contain should have the following data / properties
+
+```javasacript
```
Comment (pcraig3) [inline docs/location-setup.md]: whoops, should say `javascript` || `js`

File: docs/location-setup.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+- Optional `blocked`: CSV string containing dates that should be blocked i.e. holidays
+- Optional `checkLocationDays` function that can be used to handle custom day logic in place of the default `recurring` setup
+
+
+### Using Custom Location Day Logic (if needed)
+
+Each location can contain a custom checkLocationDays function.  This function can be used in place of the recurring property (days of the week setup).
+
+#### Example:
+```
```
Comment (pcraig3) [inline docs/location-setup.md]: could use `js` syntax highlighting here as well.

File: docs/location-setup.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+- Required String `receivingEmail`: email that will be used to send `staff` email (publicly displayed)
+- Required Object `recurring`: Used to determine available days for a location. This property takes a month property and days in the month that will be available.  
+```
+recurring: {
+    jan: ['tues', 'wed'],
+    feb: ['tues', 'wed'],
+    ...
+  },
+```
+Note: recurring can be empty object if using checkLocationDays
```
Comment (pcraig3) [inline docs/location-setup.md]: Let's say `recurring` and `checkLocationDays` with the backticks.

File: docs/location-setup.md (hunk 3 above)
Comment (pcraig3) [reply docs/location-setup.md]: could also do it on the ones above, if you think it makes them look better.

File: docs/location-setup.md (hunk 1 above)
Comment (timarney) [reply docs/location-setup.md]: 👍

**Pull request merged**